src/material-experimental/mdc-slide-toggle/slide-toggle.html

```diff
@@ -1,33 +1,50 @@
 <div class="mdc-form-field"
      [class.mdc-form-field--align-end]="labelPosition == 'before'">
-  <div class="mdc-switch mat-mdc-switch" #switch>
+  <button
+    class="mdc-switch"
+    role="switch"
+    type="button"
+    [class.mdc-switch--selected]="checked"
+    [class.mdc-switch--unselected]="!checked"
+    [tabIndex]="tabIndex"
+    [disabled]="disabled"
```

Comment:
When I tested the demo (looking at aria-required), I was able to toggle the disabled slide-toggles with the keyboard.

Reply:
I just tried it and I couldn't reproduce. Are you sure that you weren't interacting with the top slide toggle which also changes the value of the disabled one?

Reply:
Ahh, I see what happened. With VoiceOver on, screen-reader navigation moved onto the disabled toggle, but browser focus stayed on the enabled one. Nvm.